app/models/miq_shortcut.rb

```diff
@@ -20,8 +26,7 @@ def self.seed
       end
     end
 
-    all.each do |rec|
-      next if names.include?(rec.name)
+    db_data.each do |name, rec|
```

Comment:
well, that `delete` was causing duplicate records in the database. Took an approach more similar to what was there before.